tests/Unit/Evolution/Systems/Cce/Equations.py

```diff
@@ -0,0 +1,205 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+# Test function for beta equation
+
+
+def compute_integrand_for_beta(dy_j, j, one_minus_y):
```

Comment:
Could you remove the `compute_` prefixes in this file? If they didn't compute things these functions would be pretty useless ;)